test/integration/ThreeChainEndToEnd.t.sol

```diff
@@ -69,39 +68,29 @@ contract ThreeChainEndToEndDeployment is EndToEndUseCases {
     }
 
     function _setSpokes(CrossChainDirection direction) internal {
+        // NOTE: Hub is always in deployA.
         if (direction == CrossChainDirection.WithIntermediaryHub) {
-            _setSpoke(deployA, CENTRIFUGE_ID_A, sA);
```

Comment:
Why did you remove setting the spoke here? Doesn't this leave chain A without a spoke?

Reply:
It was not used and was creating confusion (at least for me) to understand the topology of the network over the test case. 

I think on one side we want to have a full ThreeChain set up, but by the other side, we have different deployments depending of the usage/directionality of the cross transfer.

Given we only need to test the cross transfer (and probably nothing else with 3 chains), I wanted to reduce the configuration overhead to mimic the test case. Tell me if you're against this and I can revert.

NOTE: also removing this spoke, helps if later I rename sB and sC to origin/target to express better from where the things go

Reply:
Thanks for the explanation! Agree it makes sense to cut the noise. Apologies for the previous state.

Reply:
Not at all! I think it has been mutated over the time